README.md

```diff
@@ -0,0 +1,15 @@
+# 🚀 Welcome to your new awesome project!
+
+This project has been created using **webpack-cli**, you can now run
+
+```
+npm run build
+```
+
+or
+
+```
+yarn build
+```
```

Comment:
While the missing README.md is not that nice, i'd suggest not to commit the one generated by `webpack-cli`.

Reply:
[977b4ab](https://github.com/demos-europe/demosplan-ui/pull/12/commits/977b4abe8329ba499056a8d1a654b91edb1af98d) true, for now I simply remove this file. Still, we might want to add a readme later